src/gui/socketapi.cpp

```diff
@@ -67,7 +67,7 @@ SocketApi::SocketApi(QObject* parent)
 
     if (Utility::isWindows()) {
         socketPath = QLatin1String("\\\\.\\pipe\\")
-        + QLatin1String("ownCloud");
+        + QLatin1String("ownCloud") + '\\' + qgetenv("USERNAME");
```

Comment:
I don't know if the username can contains non-ascii, but if that's the case I think this should be  QString::fromLocal8Bit(qgetenv("USERNAME"))